cli/docs/commands/README.md

```diff
@@ -1081,8 +1083,11 @@ FLAGS
                                 local env 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 is used by
                                 default when CLI is used in non-interactive mode
       --quiet-pull              Pull without printing progress information
+      --[no-]set-up             Set up provider, offer, commitments and deposit collateral, so there is an active offer
```

Comment:
Can you add comment about default behavior w/o this flag?

Reply:
yeah I see, can be a bit confusing. by default --set-up is set to true which not only runs docker container but also sets up provider and a working offer on the network. if you don't need it (e.g. you want to set up provider yourself using cli commands) you can pass a --no-set-up flag. Maybe it's better to just leave only a --no-set-up version cause --set-up doesn't make sence since it's already a default

Reply:
yeah, probably you need only --no-set-up